Add contributors section fixing various typos

itsmedavep opened the pull request:
Fixing typo's per #49 
Fixing #17 

Also adding in #24 

This is actually a bunch of stuff. Please review, but it might be advisable to push the change and then polish.

esaruoho commented:
also not sure where `Get Involved Here` should link to :) 

proposal, make the Kurasu into a link to http://github.com/kurasu 

also, has the "1010 patches" been updated since it now has the inigokennedy3 + nick moritz packs?

itsmedavep commented:
I have updated the Slack links to be the Slack invite link you proposed above. For now I made use the same Slack invite link. 

I updated Kurasu to be a link now.

I don't know the number of patches in total. It may exceed 1010 but I dont think that should hold up this PR. We can update after the number is settled. It seems like we are still actively soliciting and adding patches.
